firebase.json

```diff
@@ -106,7 +106,13 @@
         "source": "/dkan",
         "destination": "https://getdkan.org/",
         "type": 301
+      },
+      {
+        "source": "/estimating-worksheet",
```

Comment:
I would suggest we move this and other open practices under a common base path (probably `/open`). This will enable us to replace all of these with landing/info pages at some future date if we want, rather than having to route a bunch of top level paths.